### ISSUE TITLE
Tweak "Replace" toolbar UI for Cover blocks that use a background color instead of media

### ISSUE THREAD

**Issue by richtabor** [WordPress/gutenberg#23771]
**Is your feature request related to a problem? Please describe.**
If a user selects a color for the Cover block, the block toolbar still displays the "Replace" label for the background media control. This is a bit confusing, as its unknown what is being replaced (which is meant for media, not colors).

**Describe the solution you'd like**
Alternatives I've thought about:

1. Perhaps if you've selected a color for the Cover block, the "Replace" becomes a color picker to change the background color, and a new "add media" icon/button is present. 

2. Instead of "Replace", if a Cover block is using a background color, the label is "Add media" - or something of that nature.

<img width="610" alt="Screen Shot 2020-07-07 at 4 44 43 PM" src="https://user-images.githubusercontent.com/1813435/86840821-2bfc1680-c071-11ea-96e8-aa3440a3d068.png">

**Comment by noahshrader** [comment]
Great callout. I spent a little time with this and wanted to add a few notes.

<img width="537" alt="Screen Shot 2020-07-30 at 3 07 06 PM" src="https://user-images.githubusercontent.com/3665539/88963770-5d35c400-d276-11ea-87a1-b758b3e7d634.png">

You raise a good point here. When you set up your cover block, you choose between overlay color and background image. As you mentioned, when you choose color, the toolbar updates to include "Replace" which relates to media, which you haven't added yet (and doesn't make sense).

Curious if the attached mock would help here. Essentially adding an image icon with basic actions for media: _Add/replace/upload_ and if you have one added, _remove_. The additional settings remain in the block inspector sidebar.

**Comment by shaunandrews** [comment]
We used to have an icon for this menu, but switched to a label to make it more obvious and intuitive how to change an existing image.

I think we could move the color settings from the sidebar into the toolbar; Or, at least expose the colors settings in the toolbar when you set it from the placeholder. Also, when the block has no image the word "Replace" should probably say something like "Add media"

**Comment by richtabor** [comment]
Seems like it could be more of a Background settings UI control (for colors and media)

**Comment by noahshrader** [comment]
Awesome, yeah. I had much of the above in my first exploration but took the more consolidated route instead. Here's what I had originally, with some additional tweaks:

### Flow
![Screen Capture on 2020-08-10 at 09-43-52](https://user-images.githubusercontent.com/3665539/89789021-0c339480-daee-11ea-998a-9f06918bb5ed.gif)

The color picker popover is a quick drop to give the idea. I kept this separate from the media as these two things are accomplishing two different things: one is an overlay (color); one is a background property (media).

### Screenshots
<img width="532" alt="Screen Shot 2020-08-10 at 9 42 43 AM" src="https://user-images.githubusercontent.com/3665539/89789041-13f33900-daee-11ea-9758-8a3a25bce5d3.png">

<img width="578" alt="Screen Shot 2020-08-10 at 9 42 55 AM" src="https://user-images.githubusercontent.com/3665539/89789048-16559300-daee-11ea-9d90-86fae6b792ae.png">

**Comment by paaljoachim** [comment]
Thanks George! @Mamaduka 
You are right it looks like this has been fixed.
I will close the issue.

Using WordPress 5.7 
Gutenberg plugin 10.1.1 
Twenty Twenty One.
I see the following:

![Screen Shot 2021-03-11 at 12 15 39](https://user-images.githubusercontent.com/5323259/110779261-9a7a1f80-8263-11eb-8b9b-9593593f072d.png)

I still see we have the issue where the default text color is black inside the paragraph block in the Cover block that uses a dark color. I thought we had fixed this..... @carolinan
